src/server/CookingApp/Services/Feedback/IFeedbackService.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace CookingApp.Services.Feedback
+{
+    using CookingApp.Models.Entities;
+    using CookingApp.ViewModels.Feedback;
+
+    public interface IFeedbackService
+    {
+        Task<Feedback> CreateFeedbackAsync(CreateFeedback feedback);
+        //Task<List<Feedback>> GetFeedbacksByTitleAsync(string title);
```

Comment:
**Remove comment**

Reply:
Reason for comment: We thought that it is good for the admin to have a functionality to search for a specific feedback by title and make somthing like 'LIKE' in sql to match conditions. Removing the comment right now.